api/pom.xml

```diff
@@ -49,7 +44,6 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-test</artifactId>
-			<scope>test</scope>
```

Comment:
Jim, is the test scope definitely inherited from the parent pom as well as the version, or does it need to be repeated each time?   Thanks!

Reply:
It is (just tried) but also it can be overridden locally if needed (tried that too).

Reply:
👍  thanks for checking.